.nvmrc

```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+16
```

Comment:
Why set this to 16? FWIW Gutenberg also still uses 14.

Reply:
I will revert the node version to `14`.